esphome/components/micro_wake_word/streaming_model.cpp

```diff
@@ -175,12 +175,14 @@ VADModel::VADModel(const uint8_t *model_start, float probability_cutoff, size_t
 };
 
 bool VADModel::determine_detected() {
-  uint8_t max = 0;
+  int32_t sum = 0;
```

Comment:
`prob` is a `uint8_t`, so it doesn't make sense to have this as signed.
```suggestion
  uint32_t sum = 0;
```